src/api/two/models.rs

```diff
@@ -247,6 +257,8 @@ pub struct TransactionTypes {
 pub struct Wallet {
     pub address: String,
     pub public_key: Option<String>,
+    pub username: Option<String>,
+    pub second_public_key: Option<String>,
     #[serde(deserialize_with = "deserialize_u64_as_number_or_string")]
     pub balance: i64,
```

Comment:
@supaiku0 any reason to use `i64` instead of `u64`?

Reply:
Dunno, does serde still work when you change it to `u64`?

Reply:
Yes, @supaiku0. I've pushed a new commit with the changes (I've added a "<NOT>" to ignore it when squashing the commits")